index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="  https://developers.google.com/fonts/docs/getting_started">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="logo">
+            <a href="#">Welcome</a>         
+        </nav>
+        <nav class="union-one">
+            <a>
+                <img src="./images/Icon - Menu.png" alt="">
+            </a>
```

Comment:
[OPTIONAL] It will be great if we use a [Hamburger icon](https://fontawesome.com/icons/bars?s=solid&f=classic) here for semantics. ✔

```
<a>
  <i class="fa-solid fa-bars"></i>
</a>
```